postgresprojects.json

```diff
@@ -54,5 +54,8 @@
 	"DBA Appreciation Day": {
 		"Website": ["https://dbaday.org/"],
 		"Mastodon": ["https://mastodon.social/@dbaday"]
+	},
+	"postgres-contrib.org | Contributions to the PostgreSQL Project": {
+		"Website": ["https://postgres-contrib.org/"],
```

Comment:
Check is failing, because of the trailing `,`

Reply:
oops